src/css_parser.rs

```diff
@@ -610,6 +610,7 @@ fn parse_pixel_value<'a>(input: &'a str)
         "px" => CssMetric::Px,
         "em" => CssMetric::Em,
         "pt" => CssMetric::Pt,
+        "" if input[..split_pos] == *"0" => CssMetric::Px,
```

Comment:
I would hold off on this, simply because I don't quite agree with on the web - "if you leave off the unit, it'll be implicitly converted to px" is something that I think is going to make debugging and reading CSS harder instead of easier. The thing is that Azul CSS should be convertible to Web-CSS (to enable render-to-webpage later on), but it doesn't need to go the other way around.

That aside, what should `if input[..split_pos] == *"0"` mean? This will parse 0.0 for the value of "0", but it'll fail to parse 1.0 px for the input string of "1", for example. And I would not put the implicit conversion here, but rather return a `None` and then `.unwrap_or(CssMetric::Px)` in the `parse_pixel_value` function. This way you aren't relying on special inputs like "0".